platform/mbed_retarget.cpp

```diff
@@ -28,11 +28,15 @@
 #include "platform/mbed_critical.h"
 #include "platform/mbed_poll.h"
 #include "platform/PlatformMutex.h"
+#include "drivers/UARTSerial.h"
 #include "us_ticker_api.h"
 #include "lp_ticker_api.h"
 #include <stdlib.h>
 #include <string.h>
 #include <limits.h>
+#ifndef SSIZE_MAX
+#define SSIZE_MAX INT_MAX
```

Comment:
Should this be in mbed_retarget.h as well?

Reply:
I didn't because it currently doesn't include `<limits.h>`. Presumably it wouldn't be safe to do that before including it. Do you think I should add `<limits.h>` to it?

Reply:
I was just thinking since mbed_retarget.h declares `ssize_t`. I don't have a strong preference but imo if you needed the SSIZE_MAX someone else probably will.